### DISCUSSION TITLE
Unable to run star solo mapping, error of "an instance of 'std::out_of_range'"

### DISCUSSION THREAD

**Discussion by CampbellClark1** [alexdobin/STAR#1165] [Q&A] [answered]
Hi there,

I'm having the same issue regardless of the parameters I change with trying to run the star solo map on my data. I'm working on trout and was able to generate the genome in star solo with no issues. I've also been able to run cell ranger on my data fine, but I would much prefer to use starsolo due to the increased customization. I'm still incredibly new to analyzing single cell data and linux and I'm unable to see what is causing the issue. I've attached the script I was trying to run along with the error text and the run output log. Normally the script runs for around 1-2 minutes and then runs into the error. Is there anything obvious with what I'm doing wrong?
[star_solo_map.txt](https://github.com/alexdobin/STAR/files/6074951/star_solo_map.txt)
[Log.txt](https://github.com/alexdobin/STAR/files/6074952/Log.txt)
[error.txt](https://github.com/alexdobin/STAR/files/6074954/error.txt)

**Comment by alexdobin** [comment, marked as answer]
Hi @CampbellClark1 

have you trimmed the Read1, i.e. the barcode read? This should not be done - please try running it with the Read1 trimmed.
Also, please switch the latest release 2.7.8a.

Cheers
Alex

**Reply by CampbellClark1** [reply]
Hi Alex,

Thank you so much! That all works now, I've just been a bit stupid and new.
